### DISCUSSION TITLE
Record "Rounds" for player

### DISCUSSION THREAD

**Discussion by AidanROTF** [akiver/cs-demo-manager#759] [Ideas]
I would love for the video feature to be able to generate videos of the entire time a player is alive and auto skip to the next round when they die so I can get POV videos for my team. It seems that the feature to skip to those points exists when I click watch rounds on a player and I can obviously just record that with OBS but the video feature allows the process to be seamless and speeds up the demo so I wouldn't have to sit through most of the game 5 times to get a full set of videos. If I am missing an easy way to do this please let me know.

**Comment by akiver** [comment]
Available as of v3.3.1
